zerver/tests/fixtures/markdown_test_cases.json

```diff
@@ -294,13 +294,13 @@
       "name": "numbered_list",
       "input": "1. A\n2. B\n  3. C",
       "expected_output": "<ol>\n<li>A</li>\n<li>B<ol start=\"3\">\n<li>C</li>\n</ol>\n</li>\n</ol>",
-      "text_content": "1. A\n2. B\n 3. C"
+      "text_content": "1. A\n2. B\n  3. C"
     },
     {
       "name": "auto_renumbered_list",
       "input": "1. A\n1. B\n  1. C\n1. D",
       "expected_output": "<ol>\n<li>A</li>\n<li>B<ol>\n<li>C</li>\n</ol>\n</li>\n<li>D</li>\n</ol>",
-      "text_content": "1. A\n2. B\n 3. C\n4. D"
+      "text_content": "1. A\n2. B\n  1. C\n3. D"
```

Comment:
I guess this test case is arguably a regression?

Reply:
It's just that the original mobile notification renderer doesn't handle lists probably.